rustls/tests/unbuffered.rs

```diff
@@ -1,6 +1,10 @@
 #![allow(clippy::duplicate_mod)]
 
 use std::num::NonZeroUsize;
+
+#[cfg(feature = "withrcalias")]
+use std::rc::Rc as Arc;
```

Comment:
Maybe we should expose `alias` via `internals` instead?

Reply:
> Maybe we should expose `alias` via `internals` instead?

Should be now done & resolved thanks!